src/shared/agent_op.c

```diff
@@ -619,8 +621,17 @@ static cJSON* w_create_agent_add_payload(const char *name,
         cJSON_AddStringToObject(arguments, "id", id);
     }
 
-    if (force >= 0) {
-        cJSON_AddNumberToObject(arguments, "force", force);
+    if (force_options) {
+        cJSON* j_force = cJSON_CreateObject();
+        cJSON* j_disconnected_time = cJSON_CreateObject();
+        cJSON_AddItemToObjectCS(j_force, "disconnected_time", j_disconnected_time);
+        cJSON_AddItemToObjectCS(arguments, "force", j_force);
+        // Sending a generic configuration that replaces an agent in every possible situation
```

Comment:
From this comment I understand that we will send a hardcoded configuration with the most "permissive" configuration of force options, but, we are using the configuration as sent in **force_options**

Reply:
This comment is old, it'll be removed